plugins/resources/src/unixfilesystem.cpp

```diff
@@ -83,6 +84,80 @@
 const std::string DEFAULT_VAULT_DIR_MODE( "default_vault_directory_mode_kw" );
 const std::string HIGH_WATER_MARK( "high_water_mark" ); // no longer used
 const std::string REQUIRED_FREE_INODES_FOR_CREATE("required_free_inodes_for_create"); // no longer used
+const std::string HOST_MODE("HOST_MODE");
```

Comment:
What's the conventions around resource context string? Are they defined?

Reply:
As far as things like semicolon delimiting different keys and equal separating key from value, that is all handled in irods::parse_kvp_string() which is called in the constructor.

We did talk about case and unlike the S3 plugin the unixfilesystem plugin uses lowercase for keys so I will update it to lowercase.